scipy/optimize/tests/test_cobyla.py

```diff
@@ -71,6 +80,48 @@ def c3(w):
         assert_(sol.maxcv > 1e-6)
         assert_(not sol.success)
 
+    def test_vector_constraints(self):
```

Comment:
AFAICT, `self` is not used here, so the `setUp` is pointless for this method. You could define this test as a simple function instead.